include/quda_define.h.in

```diff
@@ -6,8 +6,8 @@
 
 /**
  * @def   __COMPUTE_CAPABILITY__
- * @brief This macro sets the target GPU architecture.  Unlike
- * __CUDA_ARCH__, this is defined on host and device.
+ * @brief This macro sets the target GPU architecture, which is
+ * defined on both host and device.
  */
 #define __COMPUTE_CAPABILITY__ @COMP_CAP@
```

Comment:
I agree that might require configuring another file but as only target specific code should depend on that this shoulf maybe move into the target specific directories.

Reply:
Similar to my comment below regarding target-specific includes not being visible outside of the QUDA build, for now this should stay here.  Once we see how this file (and quda_internal.h) looks in the context of the HIP PR we can decide how to proceed.